Makefile

```diff
@@ -53,7 +58,15 @@ LDFLAGS='-extldflags "-static"'
 controller-bin:
 	@echo '********'
 	@echo 'Compiling controller binary'
-	docker run -it --rm -v $(REPO_ROOT):$(WORK_DIR):Z -w $(WORK_DIR) -e GOOS=$(GOOS) -e GOARCH=$(ARCH) -e CGO_ENABLED=$(CGO_ENABLED) $(BUILD_IMAGE) go build -a -ldflags $(LDFLAGS) -o $(WORK_DIR)/bin/$(CONTROLLER_BIN) $(WORK_DIR)/cmd/controller/controller.go
+	$(eval DOCKER_CMD_COUNT=$(shell docker ps | wc -c))
```

Comment:
I think it is sufficient to just check if `docker ps` works. There do not have to be running containers in order for us to decide to use a containerized make.

Reply:
this does not require containers to be present, if docker is not running it evaluates correctly, if docker is not installed it evaluates correctly, and it does not need containers to be installed.